pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -176,7 +176,7 @@ private void addInstanceTagIfNeeded(String clusterName, String instanceName) {
     if (instanceTags == null || instanceTags.isEmpty()) {
       if (ZKMetadataProvider.getClusterTenantIsolationEnabled(_propertyStore)) {
```

Comment:
unrelated to this PR? what is getClusterTenantIsolationEnabled? We should not have utility methods like this in ZkMetadataProvider

Reply:
PROPERTYSTORE/CONFIGS/CLUSTER/tenantIsolationEnabled is a boolean in the cluster config. This is a helper method to get the value of that boolean. As are other methods in the same class to get stuff from propertystore.